app/common/src/buttons/Button.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 
 import * as React from 'react';
+import { Icon, type IconProps } from 'react-native-elements';
```

Comment:
Is it possible to use this? https://docs.expo.io/versions/latest/guides/icons.html